### PR TITLE
PWGHF: add possibility to apply time-frame border cut in skim creator

### PR DESCRIPTION
**Pull request by fgrosa**

### CODE REVIEW

File: PWGHF/TableProducer/trackIndexSkimCreator.cxx [hunk 1]
```diff
@@ -139,6 +141,7 @@ struct HfTrackIndexSkimCreatorTagSelCollisions {
       labels[0] = "processed";
       labels[1] = "selected";
       labels[2 + EventRejection::Trigger] = "rej. trigger";
+      labels[2 + EventRejection::TimeFrameBorderCut] = "time-frame border";
```
Comment (fcatalan92) [inline PWGHF/TableProducer/trackIndexSkimCreator.cxx:144]:
```suggestion
      labels[2 + EventRejection::TimeFrameBorderCut] = "rej. time-frame border";
```
@fgrosa Minor, I let you decide if you want to merge now or implement the change and restart the tests

**Review by fcatalan92 — DISMISSED** [review]

**Review by mfaggin — DISMISSED** [review]
thanks a lot @fgrosa

**Review by fcatalan92 — DISMISSED** [review]

File: PWGHF/TableProducer/trackIndexSkimCreator.cxx [hunk 2]
```diff
@@ -114,9 +115,10 @@ struct HfTrackIndexSkimCreatorTagSelCollisions {
   Configurable<int> nContribMin{"nContribMin", 0, "min. number of contributors to primary-vertex reconstruction"};
   Configurable<float> chi2Max{"chi2Max", 0., "max. chi^2 of primary-vertex reconstruction"};
   Configurable<std::string> triggerClassName{"triggerClassName", "kINT7", "trigger class"};
-  Configurable<bool> useSel8Trigger{"useSel8Trigger", false, "use sel8 trigger condition, for Run3 studies"};
+  Configurable<bool> useSel8Trigger{"useSel8Trigger", true, "use sel8 trigger condition, for Run3 studies"};
   Configurable<float> centralityMin{"centralityMin", 0., "Minimum centrality"};
   Configurable<float> centralityMax{"centralityMax", 100., "Maximum centrality"};
+  Configurable<float> useTimeFrameBorderCut{"useTimeFrameBorderCut", true, "use time-frame border cut in event selection"};
```
Comment (vkucera) [inline PWGHF/TableProducer/trackIndexSkimCreator.cxx:121]:
```suggestion
  Configurable<bool> useTimeFrameBorderCut{"useTimeFrameBorderCut", true, "use time-frame border cut in event selection"};
```

**Review by vkucera — CHANGES_REQUESTED** [review]

File: PWGHF/TableProducer/trackIndexSkimCreator.cxx (hunk 2 above)
Comment (fgrosa) [reply PWGHF/TableProducer/trackIndexSkimCreator.cxx]: Thanks for spotting it!

**Comment by fgrosa** [comment]
@vkucera I implemented your comment, could you please approve it now, since we cannot merge it at the moment due to your change request? Thanks!

**Review by vkucera — APPROVED** [review]

**Comment by vkucera** [comment]
> @vkucera I implemented your comment, could you please approve it now, since we cannot merge it at the moment due to your change request? Thanks!

Sorry, I forgot.

**Comment by fgrosa** [comment]
> > @vkucera I implemented your comment, could you please approve it now, since we cannot merge it at the moment due to your change request? Thanks!
> 
> Sorry, I forgot.

Thanks!

**Pull request merged**